### DISCUSSION TITLE
Error when changing the OAuth server URL.

### DISCUSSION THREAD

**Discussion by fabianostocco** [okd-project/okd#2008] [General]
Hello,

Guys, I need help with the following problem I'm facing.

OKD 4.14.0-0.okd-2024-01-06-084517.

I changed the OAuth server URL according to the documentation:
  - https://docs.openshift.com/container-platform/4.14/authentication/configuring-internal-oauth.html#customizing-the-oauth-server-url_configuring-internal-oauth.

The problem is that after the change was made, the Authentication Operator started to delete secret "v4-0-config-system-router-certs" and no longer recreate it.

I0415 17:04:44.794283       1 event.go:298] Event(v1.ObjectReference{Kind:"Deployment", Namespace:"openshift-authentication-operator", Name:"authentication-operator", UID:"a6fed5de-232d-49d3-a87a-343050721ac0", APIVersion:"apps/v1", ResourceVersion:"", FieldPath:""}): type: 'Normal' reason: 'TargetSecretDeleted' Deleted target secret openshift-authentication/v4-0-config-system-router-certs because source config does not exist
E0415 17:04:44.794887       1 base_controller.go:268] TrustDistributionController reconciliation failed: secret "v4-0-config-system-router-certs" not found
E0415 17:04:44.800526       1 base_controller.go:268] TrustDistributionController reconciliation failed: secret "v4-0-config-system-router-certs" not found
E0415 17:04:44.805338       1 base_controller.go:268] ConfigObserver reconciliation failed: secret "v4-0-config-system-router-certs" not found

I tried to create the file manually but the Operator deletes the secret again.

Em anexo logs do authentication-operator
[logsAuthenticationOperator.txt](https://github.com/okd-project/okd/files/14982320/logsAuthenticationOperator.txt)

Can you help me with this problem?

**Comment by JaimeMagiera** [comment]
Hi,

We are not working on FCOS builds of OKD any more. Please see these documents...

https://okd.io/blog/2024/06/01/okd-future-statement
https://okd.io/blog/2024/07/30/okd-pre-release-testing

We will be providing documentation on upgrading clusters from 4.15 FCOS to 4.16 SCOS. In terms of clusters that are older, you may be able to get help from community members. I'll convert this to a discussion to facilitate that.

Many thanks,

Jaime
